test/webrick/test_httprequest.rb

```diff
@@ -11,7 +11,7 @@ def teardown
 
   def test_simple_request
     msg = <<-_end_of_message_
-GET /
+GET /\r
```

Comment:
Shouldn't this be `\r\n`?

Reply:
The `\n` comes from the line ending (it's a heredoc).